### PR TITLE
Deno Deploy guide: added a note about other database types; drive-by fixes to intro wording

### PR DESCRIPTION
**Pull request by andrew-walford-prisma**
I added a note at the top to clarify that users can use this feature in conjunction with any supported database type. I also brought the intro more in line with our current style guide.

### CODE REVIEW

File: content/300-guides/200-deployment/110-deployment-guides/550-deploying-to-deno-deploy.mdx [hunk 1]
```diff
@@ -149,19 +157,19 @@ The next step is to sign up for Prisma's [Data Proxy](/data-platform/data-proxy)
...
+
    ![Import a Project](./450-03-import-project.png)
 
 4. Connect the Prisma Data Platform to your database and set up the Data Proxy:
    1. Paste your PostgreSQL connection string.
    1. Under **Location**, select a Data Proxy location that is geographically close to your database location.
    1. Click **Create Project** to test the connection and set up the Data Proxy.
-      
-   You are greeted with a new connection string that starts with `prisma://`. 
+      You are greeted with a new connection string that starts with `prisma://`.
```
Comment (janpio) [inline content/300-guides/200-deployment/110-deployment-guides/550-deploying-to-deno-deploy.mdx]: I don't think that is correct here.
image

Comment (andrew-walford-prisma) [reply content/300-guides/200-deployment/110-deployment-guides/550-deploying-to-deno-deploy.mdx]: Good catch! I'll fix it.

Comment (andrew-walford-prisma) [reply content/300-guides/200-deployment/110-deployment-guides/550-deploying-to-deno-deploy.mdx]: @janpio  This is fixed now.

**Comment by janpio** [comment]
How about we move the admonition a bit down after the first paragraph? I think it is a bit weird to start with a warning or note.

**Pull request merged**